lib/mayaUsd/ufe/UsdMaterial.cpp

```diff
@@ -26,6 +25,28 @@ PXR_NAMESPACE_USING_DIRECTIVE
 
 namespace {
 
+PXR_NS::UsdPrim getAncestorMaterial(const PXR_NS::UsdPrim& prim)
```

Comment:
This is a bug that can be fixed exclusively by removing code. You should not be adding this function.